lib/visitor.js

```diff
@@ -59,17 +151,25 @@ class Visitor {
 
     for (let i = 0; i < keyCount; ++i) {
       const key = keys[i];
+
+      // Ignore .loc.{start,end} objects.
+      if (key === "loc") {
+        continue;
+      }
```

Comment:
`loc` aren't there by default right, you have to opt-in to acorn producing them?

Reply:
That's right, but it's very important to skip them if they're there, since `loc` is a nested object (`loc.{start,end}.{line,column}`) attached to every node.

Reply:
Cool. Ya, just checking, I don't enable it and don't allow customization (to enable it).